src/modules/System/html_admin/mod_system_settings.html.twig

```diff
@@ -565,6 +570,47 @@ ZW=Zimbabwe
                     </div>
                 </div>
 
+                <div class="tab-pane fade" id="network-interface" role="tabpanel">
+                    <div class="card-header">
+                        <h3 class="card-title">{{ 'Network interface'|trans }}</h3>
+                    </div>
+                    <div class="card-body">
+                        <div class="alert {% if external_ip %}alert-success{% else %}alert-danger{% endif %}" role="alert">
+                            {% if external_ip %}
+                                {{ 'With the current settings, your FOSSBilling instance will have an external IP address of: '|trans }} {{ external_ip }}
+                            {% else %}
+                                {{ 'The currently selected network interface does not appear to be able to reach the internet!'|trans }}
+                            {% endif %}
+                        </div>
+
+                        <form class="api-form" method="post" action="{{ 'api/admin/system/set_interface_ip'|link }}" data-api-reload="1">
```

Comment:
is data-api-reload absolutely necessary? It resets the tabs, seems like if there is an issue it can report an error.

Reply:
It is not, no.
I mostly did that for testing as it's easy. (this is still a draft / WIP).

The final version can make a 2nd API call to see what the new outbound IP address is and update the page rather than refreshing the page

Reply:
The page now just updates rather than reloading